sheet2linkml/cli.py

```diff
@@ -49,30 +57,55 @@
     default=True,
     help="Controls whether we use the CCDH Terminology Service to add enumerated values for attributes.",
 )
-def main(output, filter_entity, logging_config, write_mappings, include_terminologies):
+def main(
+    google_sheet_id,
+    output,
+    filter_entity,
+    logging_config,
+    write_mappings,
+    include_terminologies,
+):
     # Display INFO log entry and up.
-    logging.config.fileConfig(logging_config)
+    if os.path.exists(logging_config):
+        logging.config.fileConfig(logging_config)
 
     # Load environemental variables from `.env` if one is present.
     load_dotenv()
 
     # Read in Google API credentials.
-    google_api_credentials = os.getenv(
-        "GOOGLE_API_CREDENTIALS", "google_api_credentials.json"
-    )
-    if not os.path.exists(google_api_credentials):
-        logging.error(
-            f"Google API Credential file '{google_api_credentials}' not found: please see "
-            + "https://github.com/cancerDHC/sheet2linkml#authorization for information on creating this file."
+    if "SHEET2LINKML_GOOGLE_SERVICE_ACCT" in os.environ:
+        # If an environment variable SHEET2LINKML_GOOGLE_SERVICE_ACCT is set, use it as a
+        # Service Account (https://pygsheets.readthedocs.io/en/stable/authorization.html#service-account)
+        pygclient = pygsheets.authorize(
+            service_account_env_var="SHEET2LINKML_GOOGLE_SERVICE_ACCT",
+            scopes=GSheetModel.SCOPES,
+        )
+    else:
+        # Otherwise, look for the path to Google API Credentials in google_api_credentials.json.
+        google_api_credentials = os.getenv(
+            "GOOGLE_API_CREDENTIALS", "google_api_credentials.json"
+        )
+        if not os.path.exists(google_api_credentials):
+            logging.error(
+                f"Google API Credential file '{google_api_credentials}' not found: please see "
+                + "https://github.com/cancerDHC/sheet2linkml#authorization for information on creating this file."
+            )
+            exit(1)
```

Comment:
Replace with `sys.exit()` as recommended in below comment?

Reply:
Fixed in 8ebf86e.